### ISSUE TITLE
Edit Profile Button overflowing

### ISSUE THREAD

**Issue by Janpreet-Singh** [mdanics/fluttergram#106]
not sure if that's the correct term to use. But, it's not completely available, 
![Edit profile button](https://user-images.githubusercontent.com/74270394/98742601-b912fb80-23d4-11eb-93de-8ecef3aa0443.jpg)
at my phone's screen

**Comment by Janpreet-Singh** [comment]
I'm a beginner to git hub and open source, recently found flutter and loving it. Any kind of assistance is highly appreciable.

**Comment by mdanics** [comment]
I think the [SafeArea](https://api.flutter.dev/flutter/widgets/SafeArea-class.html) widget would help with this

**Comment by pedr0luiz** [comment]
Can I work on this?

**Comment by iAkashPattnaik** [comment]
> I think the [SafeArea](https://api.flutter.dev/flutter/widgets/SafeArea-class.html) widget would help with this

I don't think so, see, SafeArea won't squeeze the button to the left screen witdh...

**Comment by srihariash999** [comment]
well, might be futile to comment as the project does not seem to be active, but wrapping the button with fitted box and setting the fit to scaledown should do the trick.

**Comment by dhanvi1911** [comment]
hey, could you assign this issue to me?

**Comment by Janpreet-Singh** [comment]
> hey, could you assign this issue to me?

Yeah, u can resolve it. Maybe by now someone must have fixed it, not sure as project is very old. But still you can go for it. Btw you don't have to ask anyone to assign any issues
.

**Comment by Rizwan201** [comment]
@Janpreet-Singh i would like to work on this issue

**Comment by jaykumargori** [comment]
@Janpreet-Singh You can try wrapping widget with Expanded widget to solve the issue
